stock_custom_exchange_rate_date/__manifest__.py

```diff
@@ -1,10 +1,10 @@
 {
     "name": "Customized Rate Date in Stock",
-    "version": "17.0.1.0.0",
+    "version": "18.0.1.0.0",
     "author": "Vauxoo",
     "category": "stock",
     "website": "http://www.vauxoo.com/",
-    "license": "LGPL-3",
+    "license": "OPL-1",
     "depends": [
```

Comment:
Missing LICENSE file, required by the license itself to be valid, see [this](https://git.vauxoo.com/vauxoo/project-template/-/merge_requests/114) for more info.